### PR TITLE
Cloud: commands, macros, whitelists

### PR DESCRIPTION
**Pull request by cg-tw**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [x] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/monitoring/basic-objects/commands.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+id: commandes
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/monitoring/basic-objects/commands.md:2]:
```suggestion
id: commands
```

File: cloud/monitoring/basic-objects/commands.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+      * the [mode](../../resources/glossary.md#mode) of the plugin to use.
+      * any parameter you want to pass to the plugin in this mode. For Centreon plugins, refer to the documentation of the plugin in the [Monitoring Connectors](/pp/integrations/plugin-packs/getting-started/introduction) section.
+      * You can use [macros](macros.md) (host macros or service macros) to make your command more generic, but it is not compulsory. The macros's name will appear in the configuration form for the host or service so that you can give it a specific value for each host or service you use it with.
+
+   * **Describe macros**: add a description to custom macros. This description will be visible when using the command in a host or service configuration form.
+   * **Connectors**: use the **[Perl Connector](#perl-connector)** or the **[SSH Connector](#ssh-connector)** to reduce the consumption of resources of the plugin. The Perl connector can be used with all commands except **ping** and **check_nrpe**.
```
Comment (omercier) [inline cloud/monitoring/basic-objects/commands.md:33]:
```suggestion
   * **Connectors**: use the **[Perl Connector](#perl-connector)** or the **[SSH Connector](#ssh-connector)** to reduce the consumption of resources of the plugin. The Perl connector can be used with all commands except **check_icmp** and **check_nrpe**.
```

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/monitoring/basic-objects/commands.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+      * le [mode](../../resources/glossary.md#mode) du plugin à utiliser.
+      * tout paramètre que vous voulez passer au plugin dans ce mode. Pour les plugins Centreon, reportez-vous à la documentation du plugin dans la section [Connecteurs de supervision](/pp/integrations/plugin-packs/getting-started/introduction).
+      * Vous pouvez utiliser des [macros](macros.md) (macros d'hôtes ou de services) afin de rendre votre commande plus générique, mais ce n'est pas obligatoire. Le nom de la macro apparaîtra dans le formulaire de configuration de l'hôte ou du service afin que vous puissiez lui attribuer une valeur spécifique pour chaque hôte ou service avec lequel vous l'utiliserez.
+
+   * **Description des macros** : saisissez une description pour les macros personnalisées. Cette description sera visible lorsque vous utiliserez la commande dans le formulaire de configuration d'hôte ou de service.
+   * **Connecteurs**: utilisez le **[connecteur Perl](#perl-connector)** ou le **[connecteur SSH](#ssh-connector)** afin de réduire la comsommation de ressources du plugin. Le connecteur Perl peut être utilisé avec toutes les commandes, sauf **ping** et **check_nrpe**.
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/monitoring/basic-objects/commands.md:33]:
```suggestion
   * **Connecteurs**: utilisez le **[connecteur Perl](#perl-connector)** ou le **[connecteur SSH](#ssh-connector)** afin de réduire la comsommation de ressources du plugin. Le connecteur Perl peut être utilisé avec toutes les commandes, sauf **check_icmp** et **check_nrpe**.
```

File: cloud/monitoring/basic-objects/commands.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+For security reasons, Centreon Cloud has a built-in whitelist that defines which commands are allowed to be executed by the monitoring engine of each poller. By default, it allows all commands provided by the Monitoring Connectors. If you create custom plugins with your own custom commands in it, or are using a community plugin, you will have to add their commands to the command whitelist of the poller that will run the plugin.
+
+### Add a command to the whitelist
+
+1. Log in as **root** to the poller that will run the commmand.
+2. Edit the following file: **/etc/centreon-engine-whitelist/my-whitelist.yml**.
+3. Add a line for your command, using a regex. Example: to authorize all commands from Centreon plugins, the correct syntax is **\/usr\/lib\/centreon\/plugins\/centreon_.*\.pl .***
+
+> If you have not authorized your custom command in a whitelist, the output of the plugin will say so.
```
Comment (omercier) [inline cloud/monitoring/basic-objects/commands.md:58]: C'est pas exactement l'output du plugin mais la colonne "Information" dans Resource Status

File: cloud/monitoring/basic-objects/commands.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+## Optimization connectors
+
+### SSH connector
+
+The Centreon SSH Connector is free software from Centreon available under the Apache Software License version 2 (ASL 2.0).
+It speeds up execution checks over SSH when used with Centreon Engine.
```
Comment (omercier) [inline cloud/monitoring/basic-objects/commands.md:65]:
```suggestion
It speeds up the execution and saves resources consumption by checks over SSH when used with Centreon Engine by avoiding performing all the authentication transactions at every check.
```

File: cloud/monitoring/basic-objects/commands.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+#### Installation
+
+Centreon recommends using its official packages. Most of Centreon’s endorsed software is available as RPM packages.
```
Comment (omercier) [inline cloud/monitoring/basic-objects/commands.md:69]:
```suggestion
Centreon recommends using its official packages. Most of Centreon’s endorsed software is available as RPM and DEB packages.
```

File: cloud/monitoring/basic-objects/commands.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+apt install centreon-connector-ssh
+```
+
+</TabItem>
+</Tabs>
+
+### Perl connector
+
+The Centreon Perl Connector is free software from Centreon available under the Apache Software License version 2 (ASL 2.0).
+It speeds up execution of Perl scripts when used with Centreon Engine.
```
Comment (omercier) [inline cloud/monitoring/basic-objects/commands.md:100]:
```suggestion
It speeds up execution of Perl scripts when used with Centreon Engine by avoiding compiling the Perl code at every check.
```

File: cloud/monitoring/basic-objects/commands.md [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+
+#### Installation
+
+Centreon recommends using its official packages. Most of Centreon’ endorsed software are available as RPM packages.
```
Comment (omercier) [inline cloud/monitoring/basic-objects/commands.md:104]:
```suggestion
Centreon recommends using its official packages. Most of Centreon’ endorsed software are available as RPM and DEB packages.
```

File: cloud/monitoring/basic-objects/macros.md [hunk 9]
```diff
@@ -3,96 +3,55 @@ id: macros
 title: Macros
 ---
 
-A macro is a variable used to retrieve certain values.
-A macro always starts and finishes with the “$” sign.
+A macro is a variable that defines a value to be used in a [command](commands.md) when checking hosts or services. Macros allow you to customize checks by defining a particular value for each host or service being checked by the same template: the value of the macro for a specific host of service is defined by the user in the configuration form for the host or the service.
+
+Example: In the following command (used to check the uptime on a Linux server) **$HOSTADDRESS$** is a macro.
```
Comment (omercier) [inline cloud/monitoring/basic-objects/macros.md:8]: Why not say  
```
**$HOSTADDRESS$**, **$_SERVICEWARNING$** and **$_HOSTSNMPEXTRAOPTIONS$** are macros.
```

**Review by omercier — COMMENTED** [review]
Only reviewed EN version, but I guess my comments apply to both.

**Review by omercier — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**